packages/xrpl/test/integration/transactions/payment.test.ts

```diff
@@ -12,9 +14,20 @@ const TIMEOUT = 20000
 
 describe('Payment', function () {
   let testContext: XrplIntegrationTestContext
+  let paymentTx: Payment
+  const AMOUNT = '200000000'
+  // This wallet is used for DeliverMax related tests
+  let paymentTxWallet: Wallet
```

Comment:
```suggestion
  let senderWallet: Wallet
```

Reply:
addressed in [bc803a8](https://github.com/XRPLF/xrpl.js/pull/2689/commits/bc803a8459441eddcae76919cd1e3c6283220172)